app/views/sites/index.haml

```diff
@@ -14,7 +21,7 @@
       %td= link_to site.user.name, users_path
       %td= link_to site.url, site.url
       %td= site.status.capitalize
-      %td= site.created_at.strftime '%B %e, %Y %I:%m %p'
+      %td= site.created_at.strftime '%B %e, %Y %I:%M %p'
```

Comment:
Was confused why timestamps were coming in weird? 

I saw we were using `%m` which actually shows month of the year (eg. 01..12). `%M` is what we want here to show minutes instead

More info here: https://ruby-doc.org/core-2.5.0/Time.html#method-i-strftime